client/src/components/Navigation/SigninModal.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+// import TextField from "@material-ui/core/TextField";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import { SingInModal } from "../Styles/Navigation/Signin";
+import { TextField } from "@material-ui/core";
+
+/* TODO:
+ * Create a connection to database
+ * Fill in Submit Handler
+ * Setup SignUp Model
+ * Add in Validation and Testing
+ */
+
+export default class FormDialog extends React.Component {
+  state = {
+    email: "",
+    password: "",
+    open: false
+  };
+
+  handleClickOpen = () => {
+    this.setState({ open: true });
+  };
+
+  handleSignup = () => {
+    this.setState({ open: false });
+  };
+
+  handleChange = prop => event => {
+    event.preventDefault();
+    this.setState({ [prop]: event.target.value });
+  };
+
+  handleSubmit = e => {
+    this.setState({
+      email: "",
+      password: ""
+    });
+  };
+
+  render() {
+    return (
+      <div>
+        <Button color="default" onClick={this.handleClickOpen}>
+          Sign In
+        </Button>
+        <Dialog
+          open={this.state.open}
+          onClose={this.handleClose}
+          aria-labelledby="signin"
+        >
+          <DialogContent>
+            <SingInModal>
```

Comment:
Should there be a way to escape out of the modal? At least an `x` or the ability to click outside of the region to exit. Doesn't need to be here for this but should be added in the future.

Reply:
@cpdis  Yes you're right I was working with to figure out what should happen. I was thinking of having both of our signin and signup in one modal. But for now, if you click either button it will dissapear.